_posts/0000-01-03-raghavendra98.md

```diff
@@ -1,9 +1,12 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Welcome to our new slide deck!"
```

Comment:
What's new about it?

Reply:
This new slide deck is good, so it is upgraded to new one.